pkg/db/remoteRequest_test.go

```diff
@@ -18,7 +18,7 @@ import (
 
 func TestRemoteRequestCodec(t *testing.T) {
 	cfg := config.OpenTestConfig(t)
-	cfg.ServerWhitelist = []string{"0x053b09e98ede40997546e8bb812cd838f18bb146"}
+	cfg.ServerWhitelist = []string{"0x92f91500e105e3051f3cf94616831b58f6bce1e8"}
```

Comment:
Why this needed changing?

Reply:
Changed the private key for some reason (there was a reason, I just forget it now)